spec/models/course/assessment_spec.rb

```diff
@@ -137,7 +137,7 @@
           subject.save
 
           expect(subject.folder.name).to eq(new_title)
-          expect(subject.folder.start_at).to eq(new_start_at)
+          expect(subject.folder.start_at).to be_within(0.000001).of(new_start_at)
```

Comment:
https://github.com/Coursemology/coursemology2/blob/2571b6c1d960e02afa382c0f2cbcd1a61c63a34c/spec/jobs/course/assessment/closing_reminder_job_spec.rb#L17

Saw one other spec use `.to_i` to drop the sub-second precision.

https://github.com/search?q=repo%3ACoursemology%2Fcoursemology2%20be_within&type=code

Others seem to prefer using `be_within(1.second)`, better to keep code aligned.

Reply:
understand about this. Changing to `be_within(1.second)`
The reason I did it this way is because Rails 7 is the first version of Rails in which precision for datetime is 6 by default, and hence I want to keep it as precise as possible

But then it will create a hassle since for the sake of consistency, I also need to change all the datetime to be like that. You're right that it's better to keep the code aligned with the existing way to resolve the discrepancies